README.md

```diff
@@ -117,6 +117,16 @@ owl0      IEEE 802.11  ESSID:"MyHomeWiFi"
           Power Management:on
 ```
 
+Change wifi list:
+```
+echo -n "[MyWifi_1][MyWifi_2][MyWifi_3]" | sudo tee /sys/module/vwifi/parameters/ssid_list
+```
+
+SSID Naming Convention:
+
+Don't put `]` in your SSID.
```

Comment:
Mention `[` as well.

Reply:
I think `[` is ok in my implementation though since I just split by `]` and ignore the first char which should be`[`.

Reply:
Being symmetric should make sense to the one who reads the top-level documentation.

Reply:
Mention `MyHomeWifi` as the default SSID if no explicit parameter is specified.

Reply:
Done.